.github/workflows/unpacker-image.yml

```diff
@@ -0,0 +1,34 @@
+name: build-publish-unpacker-image
+on:
+  workflow_dispatch:
+jobs:
+  build-and-publish:
+    runs-on: self-hosted
+    steps:
+      - name: Checkout Epinio
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to GitHub Docker Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and push unpacker
+        uses: docker/build-push-action@v3
+        with:
+          push: true
+          context: images/.
+          file: images/unpacker-Dockerfile
```

Comment:
Note: I looked into the [documentation of this action](https://github.com/docker/build-push-action#inputs) and saw no indication that `file` is relative to the `context`. Only the default for `file` is, i.e. when not specified.

Reply:
ok let's merge this and we will try to fix it on a branch (where we will be able to run the pipeline)

Reply:
my thought as well